controller/src/main/java/org/jboss/as/controller/ValidateModelStepHandler.java

```diff
@@ -43,7 +43,7 @@ private ValidateModelStepHandler() {
     @Override
     public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
         final Resource resource = loadResource(context);
-        if (resource == null) {
+        if (resource == null || resource.isRuntime()) {
```

Comment:
No it cannot happen, we fixed that not long a go, we make sure that if parent is runtime so is the child. I don't remember the exact commit, but it wasn't long ago. It might have been 752ccd02e6cb7ab1b292ed84f096a2c9ff8d35a9